### ISSUE TITLE
AppData Description gets rendered out of order

### ISSUE THREAD

**Issue by dahenson** [elementary/appcenter#333]
The description of my app has a paragraph block followed by an unordered list, followed by another paragraph. The unordered list somehow gets rendered below the two paragraphs in AppCenter.

My description looks like this:

`  <description>
    <p>
      Agenda is a simple, slick, speedy and no-nonsense task manager. Use it to keep track of the tasks that matter most.
    </p>
    <ul>
      <li>Blazingly fast and light</li>
      <li>Remembers your list until you clear completed tasks</li>
      <li>Autocompletion for previously added tasks</li>
      <li>Quit with the Esc key</li>
    </ul>
    <p>
      I dare you to find an easier, faster, more beautiful task manager for elementary OS.
    </p>
  </description>`

AppCenter looks like this:

![appcenterissue](https://user-images.githubusercontent.com/2007067/26949862-386f257e-4c61-11e7-9bc0-43aec0ae035b.png)

**Comment by tintou** [comment]
Okay, it's now reported upstream https://github.com/ximion/appstream/issues/116

**Comment by tintou** [comment]
TL;DR: will be probably fixed in the next libappstream

**Comment by dahenson** [comment]
Awesome! Thanks for that.

**Comment by davidmhewitt** [comment]
I'm pretty sure this got fixed at some point after reading through the related issue, please re-open if not.
